src/parse/errors.rs

```diff
@@ -446,6 +446,15 @@ impl Error {
         safe_exit(0)
     }
 
+    /// Prints the error to `stderr`
```

Comment:
More documentation please. And not always `stderr` actually.

Reply:
Is it enough good or should I implement `print_to_stderr` and `print_to_stdout`?
```
Prints the formatted and colored error message to `stdout` or `stderr` dependent on the error kind.
```

Reply:
`print` is enough I think. And change `depended on` to `according to` might be better(Oh My poor English X-) ).

Reply:
You need more documentation than that. Clap expects quite a bit of documentation for even small things. Every function doc should contain a small example conveying the usage.